filebeat/module/nats/log/test/test.log

```diff
@@ -4,6 +4,15 @@
 [1] 2019/02/06 07:19:40.624690 [INF] Server is ready
 [1] 2019/02/06 07:20:08.508891 [DBG] 172.18.0.1:38630 - cid:1 - Client connection created
 [1] 2019/02/06 07:20:08.510296 [TRC] 172.18.0.1:38630 - cid:1 - ->> [CONNECT {"verbose":false,"pedantic":false,"tls_required":false,"name":"NATS Benchmark","lang":"go","version":"1.7.0","protocol":1,"echo":true}]
-[1] 2019/02/06 07:20:08.512052 [TRC] 172.18.0.1:38630 - cid:1 - ->> [SUB foo  1]
```

Comment:
Seening this fixed here, I wonder if it is an adjustement which was made because tests didn't pass or it's just a different log?

Reply:
@ruflin Based on your comment I made an investigation. I sent to NATS slack and guys let me know that the protocol considers one or more spaces between keywords as one space. So I changed the Grok patterns accordingly.

This change was made by me when I erroneously thought that the initial double space was transferred with a typo from the original NATS logs. I reverted to be as the original with two white spaces.